plume-models/src/blogs.rs

```diff
@@ -49,6 +75,7 @@ pub struct Blog {
     pub summary_html: SafeString,
     pub icon_id: Option<i32>,
     pub banner_id: Option<i32>,
+    pub custom_domain: Option<String>,
```

Comment:
btw, i was wondering if this should be an array, and i've come to the conclusion that it should most definitely not be one